web/directory/management/commands/parse_coop_csv.py

```diff
@@ -8,10 +8,28 @@
 from yaml import load, dump
 from yaml import Loader, Dumper
 from commons.util.case_insensitive_set import CaseInsensitiveSet
-from ...services.location_service import LocationService
+# removed 8/22/22 replaced with getGeoLoc
+# from ...services.location_service import LocationService
 from django.core.management.base import BaseCommand
 from yaml import load
-  
+
+# replace LocationService with getGeoLoc
+# getGeoLoc gets data from Open Street Maps 8/22/22
+# fmi see https://www.natasshaselvaraj.com/a-step-by-step-guide-on-geocoding-in-python/
+#         https://nominatim.org/release-docs/latest/api/Overview/
+class getGeoLoc():
+    def __init__(self, zAddress=""):
```

Comment:
Instead of creating a separate class, could we put this logic into the "get_coords" method of the "LocationService" class, located at web/directory/services/location_service.py ?  Basically take lines 24 - 31 and put them into the body of LocationService.get_coords right up until "return [latitude, longitude] if latitude and longitude else None".  "get_coords" returns a latitude and longitude, so it would be great if the logic you have here replaces the logic in get_coords so it returns the proper latitude and longitude.